stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/composer/MessageComposer.kt

```diff
@@ -184,6 +192,13 @@ public fun MessageComposer(
                 )
             }
         }
+
```

Comment:
Doing this inside the `MessageComposer` `Surface`, so that its top can be used as an anchor for our suggest list popup.